src/lib/core/gestures/gesture-config.ts

```diff
@@ -50,18 +62,18 @@ export class GestureConfig extends HammerGestureConfig {
    * @returns Newly-created HammerJS instance.
    */
   buildHammer(element: HTMLElement): HammerInstance {
-    const mc = new this._hammer(element);
+    const mc = new this._hammer(element, this._hammerOptions || undefined);
```

Comment:
Add a unit test that at least has a sanity check that the provided options are applied? Doesn't need to test all the options, just that at least one of them was used.